crates/core/src/sql/mod.rs

```diff
@@ -1,4 +1,3 @@
 pub mod ast;
 pub mod compiler;
 pub mod execute;
-pub mod query_debug_info;
```

Comment:
How to annotate query metrics with details about the query will need to be reworked.